plugins/marker-droppshadow/src/index.d.ts

```diff
@@ -0,0 +1,54 @@
+import { PrepareAnimationContext } from 'vizzu/dist/plugins';
+import { Marker, Guides, MarkerLabel } from 'vizzu/dist/types/styles';
+type Next = () => void;
+export interface ShadowedMarker {
+    borderWidth?: number | null;
+    /** Opacity of the marker border. */
+    borderOpacity?: number | null;
+    borderOpacityMode?: 'straight' | 'premultiplied' | null;
+    /** Opacity of the marker fill color. */
+    fillOpacity?: number | null;
+    /** Style settings for guide lines drawn for the markers. */
+    guides?: Guides | null;
+    /** Style settings for the marker labels. */
+    label?: MarkerLabel | null;
+    /** Marker shape. */
```

Comment:
these comments are superfluous, ie. borderOpacity is quite straightforward

Reply:
I agree, i copied from lib.